themes/bootstrap3/js/facets.js

```diff
@@ -338,6 +344,7 @@ VuFind.register('multiFacetsSelection', function multiFacetsSelection() {
     const activationElem = context.querySelector('.js-user-selection-multi-filters');
     if (activationElem) {
       activationElem.addEventListener('change', function multiFacetSelectionChange() { toggleMultiFacetsSelection(this.checked); } );
+      toggleMultiFacetsSelection(localStorage.getItem(localStorageVariableName) === 'true');
```

Comment:
Since there's `saveUserSelectionLastState`, there should also be `getUserSelectionLastState` or such to retrieve the selected state (as a boolean).